gdc_client/upload/parser.py

```diff
@@ -9,15 +9,20 @@
 from . import exceptions
 
 from .client import GDCUploadClient
+from .. import log
 
 
+logger = log.get_logger('upload-client')
+
 def validate_args(parser, args):
     """ Validate argparse namespace.
     """
-    if args.manifest or args.identifier:
+    if args.manifest or args.file_ids:
+        if args.identifier:
```

Comment:
hm, if a user specifies an `identifier` but not `manifest` or `file_ids`, then the deprecation warning won't be printed, but the `'must specify either --manifest or file_id'` will?